README.md

```diff
@@ -1,181 +1,54 @@
-# Tracee
-Container and system tracing using eBPF
+# Tracee - System tracing using eBPF
 
-**Tracee** is a lightweight, easy to use container and system tracing tool.
-After launching the tool, it will start collecting traces of newly created containers (container mode) or processes (system mode).
-The collected traces are mostly system calls performed by the processes,
-but other events, such as capabilities required to perform the actions requested by the process, are also supported.
+Tracee is a lightweight and easy to use system tracing tool. It allows you to observe system calls and other system events in real time.
```

Comment:
I removed it since I thought people will expect a container tracer to be able to trace a specific container, which we can't at the moment, but I'll restore it

Reply:
https://github.com/aquasecurity/tracee/pull/70/commits/d60594e28a6d36a5a604723d412cd046cc445384